extensions/amp-instagram/1.0/storybook/Basic.amp.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {number, text, withKnobs} from '@storybook/addon-knobs';
+import {storiesOf} from '@storybook/preact';
+import {withA11y} from '@storybook/addon-a11y';
+import {withAmp} from '@ampproject/storybook-addon';
+
+// eslint-disable-next-line
+storiesOf('amp-instagram', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withA11y)
+  .addDecorator(withAmp)
+  .addParameters({
+    extensions: [{name: 'amp-instagram', version: '1.0'}],
+  })
+  .add('Default', () => {
+    return <amp-instagram shortcode="B8QaZW4AQY_" width="300"></amp-instagram>;
```

Comment:
1. Could you add knobs?
2. Use `data-shortcode` as in [current API](https://go.amp.dev/c/amp-instagram).
3. Add option for `captioned`.

```suggestion
    const dataShortcode = text('data-shortcode', 'B8QaZW4AQY_');
    const dataCaptioned = boolean('data-captioned', false);
    const width = number('width', 300);
    const height = number('height', 400);

    return (
      <amp-instagram
        data-shortcode={dataShortcode}
        data-captioned={dataCaptioned}
        width={width}
        height={height}
      ></amp-instagram>
    );
```